fractions/microprofile/microprofile-config/module.conf

```diff
@@ -0,0 +1,4 @@
+org.wildfly.extension.microprofile.config
+org.wildfly.swarm.configuration.microprofile.config export=true
+org.eclipse.microprofile.config.api export=true
+org.wildfly.extension.microprofile.config export=true
```

Comment:
Does this module need to be there twice? One with and one without `export=true`?